### PR TITLE
Remove prints from fv3core on all ranks

### PR DESCRIPTION
**Pull request by ofuhrer**
## Purpose

We should not be printing from within the dycore from all ranks. This PR removes those prints and adds some basic timestepping information to stdout for the standalone dynamical core.

## Code changes:

- `fv_dynamics.py` - print()'s removed
- `dynamics.py` - print()'s of current timestep on root rank added

## Checklist
Before submitting this PR, please make sure:

- [x] You have followed the coding standards guidelines established at [Code Review Checklist](https://paper.dropbox.com/doc/Code-Review-Checklist--BD7zigBMAhMZAPkeNENeuU2UAg-IlsYffZgTwyKEylty7NhY).
- [x] Docstrings and type hints are added to new and updated routines, as appropriate
- [x] All relevant documentation has been updated or added (e.g. README, CONTRIBUTING docs)
- [x] Unit tests are added or updated for non-stencil code changes

### CODE REVIEW

**Review by twicki — COMMENTED** [review]
lgtm, do we want the option to have the printing still? I sometimes liked it because it showed me something was going on? would putting it under `__debug__` be an option?

File: fv3core/stencils/fv_dynamics.py [hunk 1]
```diff
@@ -59,7 +59,6 @@ def compute_preamble(state, comm, grid, namelist):
 
     if namelist.hydrostatic:
         raise Exception("Hydrostatic is not implemented")
```
Comment (twicki) [inline fv3core/stencils/fv_dynamics.py:61]:
```suggestion
        raise Exception("Hydrostatic is not implemented")
    if __debug__:
        if grid.rank == 0:
            print("FV Setup")
```

**Comment by ofuhrer** [comment]
Thanks for the review @twicki . I agree with you and have changed the PR accordingly.

**Comment by ofuhrer** [comment]
launch jenkins

**Comment by rheacangeo** [comment]
There have been cases were it actual was helpful to have all ranks print out for debugging purposes, but at this stage it is fine to remove.

**Review by twicki — APPROVED** [review]
lgtm

**Pull request merged**